dashboard/test/ui/features/hamburger.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Hamburger dropdown
+
+  Scenario: Signed out user in English should not see hamburger on desktop
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    Then I wait until element "#hamburger-icon" is not visible
+
+  Scenario: Student viewing hamburger dropdown in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I set the language cookie
+    And I create a student named "Sally Student"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see ".hide-responsive-menu"
```

Comment:
Probably better to wait on a specific class/identifier rather than a "utility" CSS class like this.